addons/mail/models/mail_tracking_value.py

```diff
@@ -31,10 +32,12 @@ class MailTracking(models.Model):
 
     mail_message_id = fields.Many2one('mail.message', 'Message ID', required=True, index=True, ondelete='cascade')
 
+    track_sequence = fields.Integer('Sequence of value', readonly=1)
```

Comment:
default=100

Reply:
Tracking field sequence

Reply:
Don't forget to display the track_sequence on the mail.tracking.value form view (mail.view_mail_tracking_value_form)